src/physics/euler.cpp

```diff
@@ -632,15 +632,6 @@ ::convective_numerical_split_flux_ismail_roe(const std::array<real,nstate> &cons
     return conv_num_split_flux;
 }
 
-template <int dim, int nstate, typename real>
-real Euler<dim, nstate, real>
-::convective_surface_numerical_split_flux (
```

Comment:
Was this also a placeholder?

Reply:
I will double check with him though

Reply:
It's not even being called anywhere in the code; maybe we can remove it if its not being used?

Reply:
@AlexanderCicchino says its deprecated, I'm going to remove the function

Reply:
Okay, sounds good!

Reply:
done

Reply:
sounds good!